code/datums/discord/webhook.dm

```diff
@@ -0,0 +1,152 @@
+#define HEX_COLOR_RED 16711680
+#define HEX_COLOR_GREEN 65280
+#define HEX_COLOR_BLUE 255
+#define HEX_COLOR_YELLOW 16776960
+
+var/list/global_webhooks = list()
+
+/proc/global_initialize_webhooks()
+	var/file = return_file_text("config/webhooks.json")
```

Comment:
We generally use config settings to determine if something should be loaded from the db or from plain text.

If its set to db and there is a issue with the db connection, fall back to the plain text file

Reply:
I don't really see how something like this would benefit from being in the database.

Reply:
As with anything in the db, it can be changed much easier than hard coded config files.
I do not see a reason why there shouldnt be both options.

The code to load from the db is already there, just needs to be changed a bit.

Reply:
But this is not something that needs to be changed frequently, it's just a config option like the GELF target address.

Reply:
If the data is stored in the db (including the psks) then the API Endpoint can auth the client based on the psks stored in the db and there is no need to manually sync the psks between the endpoint and the game.

It would also allow other systems (such as the bot) to change where notifications should be sent to (something we also have right now)